boofuzz/sessions.py

```diff
@@ -1381,8 +1419,9 @@ def _post_send(self, target):
             except exception.BoofuzzTargetConnectionFailedError:
                 self._fuzz_data_logger.log_fail(constants.ERR_CONN_FAILED)
             except Exception:
-                self._fuzz_data_logger.log_fail(
-                    "Custom post_send method raised uncaught Exception." + traceback.format_exc())
+                self._fuzz_data_logger.log_error(constants.ERR_CALLBACK_FUNC.format(func_name="post_send")
+                                                 + traceback.format_exc())
+                raise exception.BoofuzzCallbackError()
```

Comment:
Raising an exception again here will cause the entire test to halt due to an error in the callback function. I originally chose to just log the test case as a failure and keep testing. The ideal choice may depend on the situation -- thoughts?

Reply:
My thoughts about this were that when an unhandled exception is raised, there is something going wrong in the callback function that shouldn't happen. Just continuing in this case might result in the user not noticing that there is a fault in the callback function.
You can still raise some of the cought exceptions above to let boofuzz know of connection problems.
Just a thought though, it's up to you how this is handled.

Reply:
@SR4ven Thanks for your thoughts. It's a close call, but in this case let's toward supporting long-running tests. Logging the error will result in an error/fail test case which should be noticeable (if it's not noticeable then the UI should be changed).

So let's take out the exception raise for now, and keep the log_error as is. Thanks!